guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-capsule-server.adoc

```diff
@@ -35,45 +35,41 @@ If you set a wildcard value `*` for the certificate's Common Name `CN =` in the
 If the command is successful, it returns two `{certs-generate}` commands, one of which you must use to generate the certificate archive file for your {SmartProxyServer}.
 ifdef::satellite[]
 +
-.Example output of `katello-certs-check`
+.To use them inside a _new-{smartproxy-example-com}_, run this command:
```

Comment:
What is `them` in this context? If it is certificates then `To use the certificates inside...` would be better. Also applies to other similar subtitles in this document.

Reply:
Addressed :+1: